src/main/java/io/github/org/programming/database/Database.java

```diff
@@ -52,11 +45,10 @@ private static void executeTableUpdate() {
             for (File file : Objects.requireNonNull(listOfFiles, "List of files is null")) {
                 if (file.isFile()) {
                     logger.info("Executing table update: {}", file.getName());
-                    statement.execute(file.getName());
+                    statement.executeUpdate(Files.readString(Path.of(file.getPath())));
                 }
             }
```

Comment:
listOfFiles ?
code handles files?

Reply:
what else should I do?

Reply:
which files are read ?

Reply:
oh, see src/main/resources/sql. The sql file and the future ones are read